core-common/src/main/java/org/glassfish/jersey/CommonProperties.java

```diff
@@ -58,6 +58,16 @@
                 "jersey.config.disableMoxyJson.server");
     }
 
+    /**
+     * Property which allows (if true) default System properties configuration provider.
+     *
+     * Effective if there are no any external properties providers
```

Comment:
"if there are no external properties providers" - the any is redundant and should not be there

Reply:
Nice spot